src/client/debugger/extension/adapter/factory.ts

```diff
@@ -112,11 +110,11 @@ export class DebugAdapterDescriptorFactory implements IDebugAdapterDescriptorFac
         return true;
     }
 
-    public getPtvsdPath(): string {
-        return path.join(EXTENSION_ROOT_DIR, 'pythonFiles', 'lib', 'python', 'new_ptvsd', 'no_wheels', 'ptvsd');
+    public getDebuggerPath(): string {
+        return path.join(EXTENSION_ROOT_DIR, 'pythonFiles', 'lib', 'python', 'new_debugpy', 'no_wheels', 'debugpy');
     }
 
-    public getRemotePtvsdArgs(remoteDebugOptions: RemoteDebugOptions): string[] {
+    public getRemoteDebuggerArgs(remoteDebugOptions: RemoteDebugOptions): string[] {
         const waitArgs = remoteDebugOptions.waitUntilDebuggerAttaches ? ['--wait'] : [];
         if (this.experimentsManager.inExperiment(DebugAdapterNewPtvsd.experiment)) {
             return ['--host', remoteDebugOptions.host, '--port', remoteDebugOptions.port.toString(), ...waitArgs];
```

Comment:
The CLI for `debugpy` is different. This has to be updated:
```
Usage: debugpy --listen [<address>:]<port>
               [--config-<name> <value>]...
               [--log-to <path>] [--log-to-stderr] [--wait-for-client]
               <filename> | -m <module> | -c <code> | --pid <pid>
```

In this case we should return.
```suggestion
            return ['--listen', `${remoteDebugOptions.host}:${remoteDebugOptions.port}`, ...waitArgs];
```

Reply:
Thanks for pointing it out. But it's weird, that when I run `python -m debugpy`, it returns me this
```
debugpy 1.0.0b1
See https://aka.ms/debugpy for documentation.

Usage: debugpy [--client] --host <address> [--port <port>]
               [--wait]
               [--no-subprocesses]
               [--log-dir <path>] [--log-stderr]
               <filename> | -m <module> | -c <code> | --pid <pid>
```

Reply:
This API in my comment is what the final one will look like. It will get pushed out today.